cartographer/mapping/collated_trajectory_builder.cc

```diff
@@ -46,10 +46,6 @@ CollatedTrajectoryBuilder::CollatedTrajectoryBuilder(
 
 CollatedTrajectoryBuilder::~CollatedTrajectoryBuilder() {}
 
-const PoseEstimate& CollatedTrajectoryBuilder::pose_estimate() const {
```

Comment:
This PR already is too large. Move the deletion of `pose_estimate()` to a follow up PR. This also allows us to change cartographer_ros without breaking it in between.